core/modules/node/node.theme.inc

```diff
@@ -176,7 +204,7 @@ function template_preprocess_node(&$variables) {
   $uri = entity_uri('node', $node);
   $variables['node_url']  = url($uri['path'], $uri['options']);
   $variables['title']     = check_plain($node->title);
-  $variables['page']      = $variables['view_mode'] == 'full' && node_is_page($node);
+  $variables['page']      = ($variables['view_mode'] == 'full' && node_is_page($node)) || (!empty($node->node_page_title));
```

Comment:
Per previous comment, this may be better to use `$node->in_preview` instead of `$node->node_page_title`.

Reply:
done